src/intTest/java/uk/nhs/digital/nhsconnect/nhais/uat/OutboundAmendmentMeshServiceUAT.java

```diff
@@ -0,0 +1,83 @@
+package uk.nhs.digital.nhsconnect.nhais.uat;
+
+import org.awaitility.Durations;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ArgumentsSource;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.web.servlet.MockMvc;
+import uk.nhs.digital.nhsconnect.nhais.IntegrationTestsExtension;
+import uk.nhs.digital.nhsconnect.nhais.jms.MeshServiceBaseTest;
+import uk.nhs.digital.nhsconnect.nhais.model.mesh.InboundMeshMessage;
+import uk.nhs.digital.nhsconnect.nhais.service.TimestampService;
+
+import java.time.ZonedDateTime;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.awaitility.Awaitility.await;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.patch;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@AutoConfigureMockMvc
+@ExtendWith(IntegrationTestsExtension.class)
+@DirtiesContext
+public class OutboundAmendmentMeshServiceUAT extends MeshServiceBaseTest {
```

Comment:
`sendToApi` in `OutboundMeshServiceUAT` already has a parameter `transactionType` that could be used to conditionally perform a patch operation instead of a post operation. Could we do that entire of creating a new test class and needing to move all of the test files to a new folder?

Reply:
Ok, duplicated class removed